test/integration/model/sync.test.js

```diff
@@ -6,6 +6,8 @@ const { DataTypes } = require('@sequelize/core');
 const expect = chai.expect;
 const Support = require('../support');
 
+const current = Support.sequelize;
```

Comment:
We're in the process of modernizing the codebase and instead of using `current` and `this.sequelize` (which are actually the same values), we're just using the `sequelize` variable

Example in this file: https://github.com/sequelize/sequelize/blob/main/test/unit/query-interface/decrement.test.ts#L19

Could you replace new usages of `current` and `this.sequelize` which just `sequelize`?

 ```suggestion
const { sequelize } = Support;
```